src/core/math.d

```diff
@@ -28,6 +28,15 @@ module core.math;
 public:
 @nogc:
 
+/*****************************************
+ * Returns x rounded to a long value using the FE_TONEAREST rounding mode.
+ * If the integer value of x is
+ * greater than long.max, the result is
+ * indeterminate.
+ */
+extern (C) real rndtonl(real x);
+
+@safe: pure: nothrow:
```

Comment:
What is the reason that `rndtonl` is not `@safe` and `nothrow`?

Reply:
Fixed.